.github/CONTRIBUTING.md

```diff
@@ -223,3 +217,43 @@ Provide some porting notes here if necessary.
 OpenZFS-issue: https://www.illumos.org/issues/1234
 OpenZFS-commit: https://github.com/openzfs/openzfs/commit/abcd1234
 ```
+
+#### Coverity Defect Fixes
+If you are submitting a fix to a Coverity defect, the commit message must
```

Comment:
We should link to the Coverity results, https://scan.coverity.com/projects/zfsonlinux-zfs